### PR TITLE
Improve `Rect` docs.

### PR DESCRIPTION
**Pull request by frewsxcv**
Fixes https://github.com/georust/geo/issues/441.

### CODE REVIEW

File: geo-types/src/rect.rs [hunk 1]
```diff
@@ -1,6 +1,25 @@
 use crate::{polygon, Coordinate, CoordinateType, Polygon};
 
-/// A bounded 2D quadrilateral whose area is defined by minimum and maximum `Coordinates`.
+/// A bounded 2D quadrilateral whose area is defined by minimum and maximum `Coordinate`s.
+///
+/// The constructors and setters ensure the maximum `Coordinate` is greater than or equal to the
+/// minimum. Thus, a `Rect`s width, height, and area is guaranteed to be greater than or equal to
+/// zero.
```
Comment (michaelkirk) [inline geo-types/src/rect.rs:7]: 👍

**Comment by frewsxcv** [comment]
bors r=michaelkirk

**Comment by frewsxcv** [comment]
bors r=michaelkirk

**Comment by frewsxcv** [comment]
bors r=michaelkirk

**Pull request merged**